Makefile

```
@@ -100,9 +100,13 @@ public/packs/manifest.json: yarn.lock $(shell find app/javascript -type f) ## Bu
 	yarn build
 
 test: export RAILS_ENV := test
-test: $(CONFIG) ## Runs RSpec and yarn tests
+test: $(CONFIG) ## Runs RSpec and yarn tests in parallel
 	bundle exec rake parallel:spec && yarn test
 
+sync_test: export RAILS_ENV := test
+sync_test: $(CONFIG) ## Runs RSpec and yarn tests synchronously
+	bundle exec rake spec && yarn test
```

maybe "serially" instead of synchronously?

```suggestion
test_serial: export RAILS_ENV := test
test_serial: $(CONFIG) ## Runs RSpec and yarn tests serially
	bundle exec rake spec && yarn test
```

---

Sure, in the end it's all threads which I have on the brain :)